.github/workflows/autobuild.yaml

```diff
@@ -90,6 +94,33 @@ jobs:
             echo "$($name)=$($value)" >> $env:GITHUB_ENV
           }
         }
+    # FIXME: gRPC support is currently available only on Linux. Other platforms
+    # will be added later.
+    - name: Clone gRPC
+      if: matrix.config.name == 'linux'
+      uses: actions/checkout@v2
+      with:
+        repository: grpc/grpc
+        path: grpc
+        ref: v1.32.0
+        submodules: recursive
+    - name: Prepare gRPC build environment
+      if: matrix.config.name == 'linux'
+      run: |
+        mkdir grpc-installation
```

Comment:
nit: I would merge these with the next step. Also you don't need the environment variables as `cwd` doesn't change throughout the script.